src/sentry/rules/processing/processor.py

```diff
@@ -10,8 +10,7 @@
 from django.core.cache import cache
 from django.utils import timezone
 
-from sentry import analytics, features
-from sentry.buffer.redis import RedisBuffer
```

Comment:
@wedamija is this just moving when we create the connection to redis to application boot time? if so, do we need to do anything additional to make sure we're maintaining a specific redis connection?

Reply:
The problem is that usually when we create the buffer we do it here: https://github.com/getsentry/sentry/blob/94c896a4a3663abbd31775957f1aa5448fde5491/src/sentry/buffer/__init__.py#L7

When we do this, we also pass options along with it. In prod we pass these: https://github.com/getsentry/getsentry/blob/d7c44c232d49584b17e7ef747b77b145969d3ec6/getsentry/conf/settings/prod.py#L1084-L1085

```
SENTRY_BUFFER = "sentry.buffer.redis.RedisBuffer"
SENTRY_BUFFER_OPTIONS = {"cluster": "buffer"}
```

So it points us to a specific cluster. When we manually instantiate, we don't pass those options. In general, it's better to use `buffer` so that we respect our customer configuration